compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs

```diff
@@ -158,7 +165,7 @@ contextLocation = \case
   ContextTemplate _ t _  -> tplLocation t
```

Comment:
this made me sad: even if the `TemplatePart` has a location somewhere, the error message only shows the location of the template body